llvm/include/llvm/CodeGen/VirtRegMap.h

```diff
@@ -32,9 +32,7 @@ class TargetInstrInfo;
 
   class VirtRegMap : public MachineFunctionPass {
   public:
-    enum {
-      NO_STACK_SLOT = (1L << 30)-1,
-    };
+    static constexpr unsigned NO_STACK_SLOT = (1u << 30) - 1;
```

Comment:
judging by the return type of `getStackSlot` this should be `int` instead?

Reply:
While on it maybe remove this `public:` section here and merge it with the one below?

Reply:
> judging by the return type of `getStackSlot` this should be `int` instead?

Yep. I had unsigned on the brain, then changed it but forgot to commit it.